compiler/rustc_expand/src/mbe/macro_rules.rs

```diff
@@ -1025,7 +1025,7 @@ fn check_matcher_core(
                             err.span_label(sp, format!("not allowed after `{}` fragments", kind));
 
                             if kind == NonterminalKind::PatWithOr
-                                && sess.edition == Edition::Edition2021
+                                && sess.edition >= Edition::Edition2021
```

Comment:
why does this not use `rust_2021`?

Reply:
Fixed.

Reply:
Actually, no it's not fixed. Turns out those methods don't exist on that type (there's multiple `Edition` types from different crates). I'll add those methods.